### PR TITLE
fix: fixing docs css reload

### PR DESCRIPTION
**Pull request by helendurranttw**
☝️ make the title meaningful and follow conventional commits standards

## ✅ Checklist

- [ ] All changes are covered by tests
- [ ] All changes have been crossbrowser checked, especially IE11
- [ ] The changes are covered in docs

### CODE REVIEW

File: packages/css/src/less/components/_component-animations.less [hunk 1]
```diff
@@ -1,7 +1,8 @@
 // Component animations
 .fade {
+  transition: opacity 0.15s linear;
```
Comment (helendurranttw) [inline packages/css/src/less/components/_component-animations.less:3]: These mixins just added vendor prefixes.

File: packages/components/.gulp.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "flags": {
+    "gulpfile": "../../node_modules/@transferwise/less-config/gulpfile.js"
```
Comment (andreapiras) [inline packages/components/.gulp.json:3]: Very nice

Comment (andreapiras) [reply packages/components/.gulp.json:3]: Shall we maybe mention this config inside less-config package README?

File: yarn.lock [hunk 3]
```diff
@@ -9,6 +9,14 @@
   dependencies:
     cross-fetch "3.0.4"
 
+"@ampproject/toolbox-core@^2.5.0":
```
Comment (dozortsev) [inline yarn.lock:12]: where this dependencies coming from?

Comment (helendurranttw) [reply yarn.lock:12]: Good point, I think I reverted some deps but not the lock file. I'll revert this.

**Pull request merged**